privval/priv_validator.go

```diff
@@ -174,21 +226,21 @@ func (pv *FilePV) SignProposal(chainID string, proposal *types.Proposal) error {
 
 // returns error if HRS regression or no LastSignBytes. returns true if HRS is unchanged
 func (pv *FilePV) checkHRS(height int64, round int, step int8) (bool, error) {
```

Comment:
This would be nicer as a method on the LastSignState directly, then we wouldn't need to reference it so many times. Alternatively we could say `lss := pv.LastSignState` and use `lss` instead of `pv.LastSignState` everywhere